pkg/virt-config/config-map.go

```diff
@@ -311,6 +311,7 @@ type migrationConfiguration struct {
 	UnsafeMigrationOverride           *bool              `json:"unsafeMigrationOverride,string,omitempty"`
 	AllowPostCopy                     *bool              `json:"allowPostCopy,string,omitempty"`
 	DisableTLS                        *bool              `json:"disableTLS,omitempty"`
+	DedicatedMigrationNetwork         *string            `json:"dedicatedMigrationNetwork,omitempty"`
```

Comment:
I think you can remove this. We don't add new options to the config-map anymore. If people would try to set it, we would just ignore it and stick with the default.

Reply:
I wish! The project won't compile if this struct is not identical to the CR struct.

Reply:
Ack. With https://github.com/kubevirt/kubevirt/pull/6784 it will not have a relevance anymore.